packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -40,7 +43,7 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         const messagingContainer = options.paypalCommerce?.messagingContainer;
         const isMessagesAvailable = Boolean(messagingContainer && document.getElementById(messagingContainer));
 
-        await this._paypalCommercePaymentProcessor.initialize(this._getParamsScript(initializationData, cart));
+        await this._paypalCommercePaymentProcessor.initialize(this._getParamsScript(initializationData, cart), undefined, undefined, initializationData.isVenmoEnabled);
```

Comment:
Does this seem right that we are passing quite a few `undefined` here?

Reply:
Yes, we will refactor it later after adding separate strategies for each payment method